docs/tendermint-core/secure-p2p.md

```diff
@@ -3,10 +3,6 @@
 The Tendermint p2p protocol uses an authenticated encryption scheme
 based on the [Station-to-Station
 Protocol](https://en.wikipedia.org/wiki/Station-to-Station_protocol).
-The implementation uses
```

Comment:
We should say what we use instead and link to some discussion around it. It's xchacha20poly1305 instead of nacl box's xsalsa20poly1305